src/core/geometry/qgsgeometrycollection.h

```diff
@@ -204,6 +204,15 @@ class CORE_EXPORT QgsGeometryCollection: public QgsAbstractGeometry
     //! Adds a geometry and takes ownership. Returns TRUE in case of success.
     virtual bool addGeometry( QgsAbstractGeometry *g SIP_TRANSFER );
 
+    /**
+     * Adds a list of geometries to the collection, transferring ownership to the collection.
+     *
+     * Returns TRUE in case of success.
```

Comment:
\returns instead?

Reply:
![image](https://github.com/qgis/QGIS/assets/1829991/57f20c21-a867-44e7-a3e7-57d75a190a95)

:rofl:

Reply:
(I was going to suggest we address that in bulk in a follow up, but I'm inclined not to if there's that many hits!)

Reply:
:scream: 

Alright, it's something that should be addressed (if it is to be changed) in a different PR.